test/source/GetTypeName.h

```diff
@@ -81,7 +81,12 @@ eastl::string GetTypeName()
 			const char* pDemangledName = NULL;
 		#endif
 
-		if(pDemangledName && pDemangledName[0])
+		// Gets rid of pointer to bool conversion warning when pDemangledName is an array.
+		#if EASTL_LIBSTDCPP_DEMANGLE_AVAILABLE
+			if(pDemangledName && pDemangledName[0])
```

Comment:
I'd prefer to not include this delta as the added code complexity isn't required to optimize away an additional comparison.